templates/cluster/openstack-standalone-cp/values.yaml

```diff
@@ -0,0 +1,89 @@
+# cluster parameters
+controlPlaneNumber: 3
+workersNumber: 2
+
+clusterNetwork:
+  pods:
+    cidrBlocks:
+    - "10.244.0.0/16"
+  services:
+    cidrBlocks:
+    - "10.96.0.0/12"
+  serviceDomain: "cluster.local"
+
+# openstack cluster parameters
+openstack:
+  authURL: "" # Keystone URL
+  region: "" # OpenStack region
+  username: "" # OpenStack username
+  password: "" # OpenStack password
+  useApplicationCredentials: false # set to true if using application credentials instead of username/password
+  applicationCredentialID: "" # OpenStack application credential ID (if applicable)
+  applicationCredentialSecret: "" # OpenStack application credential secret (if applicable)
+
+clusterIdentity:
+  name: ""
+  cloudName: ""
+  region: ""
```

Comment:
Since this is not cluster identity object reference (as in other providers) I suggest renaming it (to `identityRef` for example) to avoid confusion.
P.S. This topic probably should be brought up in a larger context.

Reply:
Changed to identityRef.